setup/mail-postfix.sh

```diff
@@ -93,6 +93,12 @@ tools/editconf.py /etc/postfix/master.cf -s -w \
 # Install the `outgoing_mail_header_filters` file required by the new 'authclean' service.
 cp conf/postfix_outgoing_mail_header_filters /etc/postfix/outgoing_mail_header_filters
 
+# Modify the `outgoing_mail_header_filters` file to use the local machine name and ip 
+# on the first received header line.  This may help reduce the spam score of email by
+# removing the 127.0.0.1 reference.
+sed -i "4s/unknown/$PRIMARY_HOSTNAME/" /etc/postfix/outgoing_mail_header_filters
+sed -i "4s/127.0.0.1/$PUBLIC_IP/" /etc/postfix/outgoing_mail_header_filters
+
```

Comment:
Likewise, please change `127.0.0.1` in `conf/postfix_outgoing_mail_header_filters` (and here) to `PUBLIC_IP`.